tests/simulation/params/test_validators_params.py

```diff
@@ -87,7 +87,7 @@ def test_low_mach_preconditioner_validator(
     assert params
 
     message = (
-        "Low-Mach preconditioner output requested, but low-Mach preconditioner mode is not enabled."
```

Comment:
There are complaints from MC that the error message is not clear on how user should correct the settings. Let's try to have a more detailed message.

Reply:
See next comment